src/encoded/tests/features/conftest.py

```diff
@@ -29,7 +29,7 @@ def app(app_settings):
 
 @pytest.mark.fixture_cost(500)
 @pytest.yield_fixture(scope='session')
-def workbook(request, app):
+def index_workbook(request, app):
```

Comment:
How does renaming help?

Reply:
prevents confusion between the workbook fixture that doesn't index in ES and the one that does